scripts/quests/otherAreas/The_Gift.lua

```diff
@@ -0,0 +1,93 @@
+-----------------------------------
+-- The Gift
+-----------------------------------
+-- Log ID: 4, Quest ID: 21
+-- Oswald  : !pos 47.119 -15.273 7.989 248
+-----------------------------------
+require('scripts/globals/items')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.THE_GIFT)
+
+quest.reward =
+{
+    item = xi.items.SLEEP_DAGGER,
+    title = xi.title.SAVIOR_OF_LOVE
```

Comment:
Does these quests offer any selbina fame?

Reply:
I don't see any fame rewarded in the original code and I don't see any reference to fame rewards from what I can find.

Reply:
oh! I thought all quests rewarded fame if they were apart of a fame area. My mistake.

Reply:
I won't be surprised if it does but I'm not sure where to find info as to whether or not it does and how much.